CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to SeedLang
+
+Thanks for taking the time to contribute!
+
+## Design decisions
+
+There are lots of trade-offs to made when building a new programming
+environment. We will document every significant design decision in the `design`
+dir. Please create a new issue labeled with `design` if you find anything
+undocumented or propose an update for an existing design.
+
+## Submitting changes
```

Comment:
Do we need any rule for the commit message?

Reply:
Will add a short description to encourage contributors to follow the guideline at https://chris.beams.io/posts/git-commit/